node/src/node.rs

```diff
@@ -74,6 +75,8 @@ where
     pub p2p_listen_on: Vec<Multiaddr>,
     /// Maximum number of headers in batch while syncing.
     pub sync_batch_size: u64,
+    /// Syncing window size, pruning starts one hour after syncing window end
```

Comment:
same here, please add shortly what syncing window is

Reply:
also I'm wondering if we could use Option there to automatically choose default one in case it's not provided by user. But no hard opinion, will defer to @oblique :p

Reply:
I agree with `Option` but what `None` means in this case? Maybe some users will think that it will disable syncing window entirely. Maybe `custom_syncing_window: Option<Duration>` will make more sense.

Reply:
We need to revisit builder pattern.

Reply:
Switched to custom_syncing_window and updated the comment. Plus one on the builder pattern xd